### ISSUE TITLE
Example from readme not working

### ISSUE THREAD

**Issue by lukasz-golebiewski** [gnieh/diffson#7]
The example on the main page, when run yields:
```
gnieh.diffson.PatchException: element a does not exist in "{\n  \"a\": 1,\n  \"b\": true,\n  \"c\": \"test\"\n}"
	at gnieh.diffson.Operation.action(my.sc0.tmp:136)
	at gnieh.diffson.Replace.action(my.sc0.tmp:230)
	at gnieh.diffson.Operation.apply(my.sc0.tmp:112)
	at gnieh.diffson.JsonPatch$$anonfun$apply$1.apply(my.sc0.tmp:42)
	at gnieh.diffson.JsonPatch$$anonfun$apply$1.apply(my.sc0.tmp:41)
	at scala.collection.LinearSeqOptimized$class.foldLeft(my.sc0.tmp:120)
	at scala.collection.immutable.List.foldLeft(my.sc0.tmp:80)
	at gnieh.diffson.JsonPatch.apply(my.sc0.tmp:41)
	at gnieh.diffson.JsonPatch.apply(my.sc0.tmp:49)
	at #worksheet#.json3$lzycompute(my.sc0.tmp:21)
```
```
import gnieh.diffson._

val json1 = """{
              |  "a": 1,
              |  "b": true,
              |  "c": "test"
              |}""".stripMargin
val json2 = """{
              |  "a": 6,
              |  "c": "test2",
              |  "d": false
              |}""".stripMargin
val json3 = JsonDiff.diff(json1, json2)(json1)
println(json3)
```

**Comment by satabin** [comment]
You are right, the string variant of the `apply` method in `JsonPatch` is not selected anymore since I switched to spray-json.

**Comment by lukasz-golebiewski** [comment]
Are you going to release this bugfix any time soon?

**Comment by satabin** [comment]
I just published a snapshot version (1.0.0.-SNAPSHOT) with this fix.
I think the next release will be next week.
